docs/configuration-reference-sink.md

```diff
@@ -0,0 +1,97 @@
+# Database
+
+## `cloudant.url`
+Cloudant database connection URL (eg `https://<uuid>.cloudantnosqldb.appdomain.cloud`)
+
+* Type: `string`
+* Valid Values: `<any URL>`
+
+## `cloudant.db`
+Cloudant database name (for sink connector it will be created if it does not exist)
+
+* Type: `string`
+
+# Kafka
+
+## `topics`
+Kafka topic list
+
+* Type: `list`
+
+## `batch.size`
+Size of batches to send to Cloudant _bulk_docs endpoint
+
+* Type: `int`
+* Default: `1000`
+* Valid Values: `[1,...,20000]`
```

Comment:
This is in error, it should be capped at 2k, I didn't spot [the constant](https://github.com/IBM/cloudant-kafka-connector/blob/31689759367333110435e0b0111f1dedb564a560/src/main/java/com/ibm/cloud/cloudant/kafka/utils/InterfaceConst.java#L46) being wrong in the PR that introduced it, sorry.

Reply:
Should I fix the ConfigDef in this PR? I just realised that https://github.com/IBM/cloudant-kafka-connector/pull/114 would have been the ideal place to do this but I've merged it.

Reply:
Yeah I'm ok with an additional commit in this PR for the change.
I mean on some level you'd need the fix here anyway to regenerate the docs, but I suspect in practice it is simpler to just delete the extra `0`, but I think we can justify the change being loosely associated with this work.